### DISCUSSION TITLE
Moor not finding type converter

### DISCUSSION THREAD

**Discussion by akwasiio** [simolus3/drift#1103] [Q&A] [answered]
@simolus3 I get this error whenever I try to build my app, any idea what I may be doing wrong? 

```
Error: Method not found: 'UserTypeConverter'.
      static TypeConverter<User, String> $converter0 = const UserTypeConverter();
```

**Comment by simolus3** [comment, marked as answer]
Can you import the library defining `UserTypeConverter` into the library defining your database class? Unfortunately we can't generate imports in part files, so you need to do that manually.

**Reply by akwasiio** [reply]
It worked! Thank you

**Comment by ShaalanMarwan** [comment]
is there any update on this ? I would split it into files and I want to generate it automatically.

**Reply by simolus3** [reply]
You can use the new [modular code generation](https://drift.simonbinder.eu/docs/generation-options/modular/) mode to make drift emit independent files with their own imports.
